### ISSUE TITLE
Excluding objects from light sources

### ISSUE THREAD

**Issue by CrossroadInTheVoid** [3dreamengine/3DreamEngine#70]
Is there any way to exclude objects from light source?
I've noiced that at least in old versions of 3DreamEngine SpriteBatches are affected by light sources and when they have their own emission, they become too bright.

**Comment by Luke100000** [comment]
Yes, no light equals black albedo color, so you can set the color to black.

For example a candle flame:
```lua
local spriteBatch = dream:newSpriteBatch(texture, texture)
spriteBatch.material.color = {0, 0, 0, 1.0}
```

The same emission texture is used for albedo, but we effectively only use the alpha channel.
